lib/Importer/ONNXModelLoader.cpp

```diff
@@ -3207,6 +3207,58 @@ Error ONNXModelLoader::loadCmpLTE(const ONNX_NAMESPACE::NodeProto &op,
   return Error::success();
 }
 
+static Error getShapeForBroadcast(llvm::ArrayRef<NodeValue> inputs,
```

Comment:
Can you add a doxygen comment explaining what this function does? E.g. `takes a list of NodeValues \p inputs and broadcasts them to a common shape based on the maximum dimension of ...`

Reply:
Done.